results.tex

```diff
@@ -415,6 +423,21 @@ \subsection{Solver Comparisons}
 \end{tabular}
 \end{table}
 
+Each of the results in Table \ref{tbl:timing} is with respect to the simple
+scenarios presented in this paper. In order to ascertain a sense of how each
+solver will work with larger scenarios, the \basecase scenario was run an
+additional time after modifying the reactor deployment schedule as follows: in
+each instance when a single reactor would be deployed, five reactors are
+deployed instead, increasing the total reactor population by a factor for five
+to approximately 150. The greedy and cbc \basecase scenarios solved in 10.7 and
+20.8 seconds, respectively. This implies that there is not a clear linear
+relationship between solution time of a full simulation run and the number of
+entities in the simulation for either solver -- there are likely additional
+simulation dynamics occurring outside of the DRE that affect solution
+times. Additionally, this simple exercise shows that simulations of reactor
+populations approximately the size of the United States reactor fleet can be
+solved in very reasonable times.
```

Comment:
I think this is a good addition.  It's not necessarily "more complex" but I don't think we can introduce a more complex scenario without adding a lot of description of that scenario, and it would be distracting.